localization/zh_TW.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+$labels = Array(
+	'cd_title' => 'CardDAV',
+	'cd_active' => '激活CardDAV地址簿',
+	'cd_newabboxtitle'=> "新增通訊錄",
+	'cd_username' => '用戶名',
+	'cd_password' => '密碼',
+	'cd_url' => 'URL',
+	'cd_name' => "通訊錄名稱",
+	'cd_use_categories' => "創建新的聯繫人組時，使用類別而不是KIND = group（RFC6350）\n",
+	'cd_delete' => "刪除此通訊錄",
+	'cd_name_new' => "配置新地址簿",
+	'cd_refresh_time' => "更新間隔（小時）",
+	'cd_err_noabfound' => '找不到通訊錄',
+	'cd_enabled' => '已啟用',
+	'cd_disabled' => '已禁用',
+	'cd_lastupdate_time'=>'最後更新時間',
+	'cd_resync' => "強制與服務器立即同步",
+	'cd_msg_synchronized' => 'CardDAV地址簿$ name同步（花費 $duration秒）',
```

Comment:
There is a space in `$ name` that will prevent proper expansion of the variable. Please remove the space.

Reply:
done